test/activations.jl

```diff
@@ -332,39 +320,57 @@ end
         gradtest(f, +2 .+ rand(rng, 2, 2), check_broadcast=true)
         gradtest(f, -2 .- rand(rng, 2, 2), check_broadcast=true)
 
-        if Symbol(f) in NNlib.UNARY_ACTS
+        if f in UNARY_FUNCTIONS
             @test rrule(f, rand()) !== nothing
             @test rrule(broadcasted, f, rand(2)) !== nothing
         end
-        if Symbol(f) in NNlib.BINARY_ACTS
-            @test rrule(f, rand(), rand()) !== nothing
-            @test rrule(broadcasted, f, rand(2), rand(2)) !== nothing
```

Comment:
I think these tests were never run. Because NNlib.BINARY_ACTS contains tuples, not symbols.